interface/modules/save_items.rb

```diff
@@ -0,0 +1,48 @@
+require 'json'
+require './classes/music_album'
+require './classes/book'
+require './classes/game'
+
+module SaveItems
+  def save_music_album
+    File.open('./data/music_album.json', 'w') do |file|
+      file.puts @music_album.map { |album|
```

Comment:
- [ ] Good job adding `module SaviItems` 😄, however, you need to fix one more thing, In your video presentation, I saw that you had initially a `data` folder, then after cloning your project, I couldn't see it because you deleted it 😢 
![Screenshot 2023-10-20 003725](https://github.com/ivangonzalez224/Catalog-of-my-things/assets/57408419/02de1638-8ae9-46c8-96af-87f243a8bb02)

So if you run the project, we will not be able to Preserve the data because it gives an error 

![Screenshot 2023-10-20 001644](https://github.com/ivangonzalez224/Catalog-of-my-things/assets/57408419/d0f0b816-9981-40e2-a4d4-7b94b871f93a)

But let's just create it (`data` folder) and then rerun the project, now it will preserve the data

![Screenshot 2023-10-20 001741](https://github.com/ivangonzalez224/Catalog-of-my-things/assets/57408419/bcef1850-1e82-43ee-9abf-c589e1d7998b)
 
Kindly fix it 👍

Reply:
Fixed.